conda/multisheller/ergocub-software_activate.msh

```diff
@@ -0,0 +1,27 @@
+if_(is_set("COMSPEC")).then_([
+	sys.list_append("YARP_DATA_DIRS", path.join(env("CONDA_PREFIX"), "Library\\share\\ergoCub")),
+	sys.list_append("ROS_PACKAGE_PATH", path.join(env("CONDA_PREFIX"), "Library\\share")),
+	sys.list_append("AMENT_PREFIX_PATH", path.join(env("CONDA_PREFIX"), "Library")),
+	sys.list_append("GAZEBO_MODEL_PATH", path.join(env("CONDA_PREFIX"), "Library\\share\\gazebo\\models")),
+	sys.list_append("GAZEBO_RESOURCE_PATH", path.join(env("CONDA_PREFIX"), "Library\\share\\gazebo\\worls"))
```

Comment:
I do not think you the ergocub-software package is installing anything in the `<install_prefix>/share/gazebo` folder, right? So probably we can just remove these lines?

Reply:
I removed them also in the `_deactivate` file